tests/python/utils.py

```diff
@@ -417,6 +418,35 @@ def inner_fn(*args, **kwargs):
     return inner_fn
 
 
+UPDATED_SDPA = LooseVersion(torch.__version__) >= LooseVersion("2.7.0")
+
+
+def get_sdpa_rng_nvf_tensors(fd: FusionDefinition):
+    dtype = DataType.UInt64 if UPDATED_SDPA else DataType.Int
+    is_cpu = False if UPDATED_SDPA else True
+    philox_shape = [-1] if UPDATED_SDPA else []
```

Comment:
```suggestion
    philox_shape = [2] if UPDATED_SDPA else []
```
We know the size must be 2 so why making our life harder?